lib/quibble.mjs

```diff
@@ -6,49 +6,63 @@ export const ignoreCallsFromThisFile = quibble.ignoreCallsFromThisFile
 export const config = quibble.config
 
 export async function resolve (specifier, context, defaultResolve) {
```

Comment:
Does this function still need to be async with the removal of `await`?

Reply:
There's as yet no formal spec for hooks, but the example in the documentation shows it to be an async function, so I'm assuming it has to return a `Promise`, which is easiest to do by making it `async`. But you're right that there's no _internal_ reason to do so (although I don't understand why they didn't make `defaultResolve` async).

https://nodejs.org/api/esm.html#esm_code_resolve_code_hook